x/ccv/provider/client/cli/tx.go

```diff
@@ -176,7 +176,7 @@ func NewSubmitConsumerDoubleVotingCmd() *cobra.Command {
 				return err
 			}
 
-			msg, err := types.NewMsgSubmitConsumerDoubleVoting(submitter, ev, nil)
+			msg, err := types.NewMsgSubmitConsumerDoubleVoting(submitter, ev, &header)
```

Comment:
This change was already introduced in a [previous PR](https://github.com/cosmos/interchain-security/pull/1247/files). Idnk why it strangely appears here as well.